packages/tools/webpack-fluid-loader/package.json

```diff
@@ -94,7 +94,7 @@
 	"devDependencies": {
 		"@fluid-tools/build-cli": "^0.19.0-165129",
 		"@fluid-tools/webpack-fluid-loader-previous": "npm:@fluid-tools/webpack-fluid-loader@2.0.0-internal.5.0.0",
-		"@fluidframework/build-common": "^1.2.0",
+		"@fluidframework/build-common": "^2.0.0-151899",
```

Comment:
Do you want to release build-common before you merge? We'll have to do it anyway so might as well just do it now. That way you can just integrate 2.0 directly instead of the prerelease. Your call.

Reply:
Can we do release from next branch?

Reply:
Will do it from main since the changes are already integrated into main

Reply:
OPened PR here with 2.0.0 since I got several merge conflicts here.
https://github.com/microsoft/FluidFramework/pull/15941